spec/post_integraion_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rails_helper'
+
+RSpec.configure do |config|
+  config.before(:each, type: :system) do
+    driven_by :selenium, using: :chrome, options: {
+      binary: '/usr/bin/google-chrome' # Replace with the actual Chrome binary location on your Chromebook
+    }
+  end
+end
+
+RSpec.describe 'Post', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:user1) do
+    User.create(name: 'John', photo: 'https://media.giphy.com/media/tZaFa1m8UfzXy/giphy.gif',
+                bio: 'This is John\'s bio', posts_counter: 0)
+  end
+
+  before do
+    @user2 = User.create(name: 'Lily', photo: 'https://media.giphy.com/media/K4x1ZL36xWCf6/giphy.gif',
+                         bio: 'This is Lily\'s bio', posts_counter: 0)
+    Post.create(author: user1, title: 'Name of the wind', text: 'The book of wind.', comments_counter: 0,
+                likes_counter: 0)
+    @post2 = Post.create(author: user1, title: 'Name of the fire', text: 'The book of fire.', comments_counter: 0,
+                         likes_counter: 0)
+    Post.create(author: user1, title: 'Name of the earth', text: 'The book of earth.', comments_counter: 0,
+                likes_counter: 0)
+    Post.create(author: user1, title: 'Name of the water',
+                text: 'The book of water and book of water book of water book of water book of water book of water' \
+                      'book of water book of water book of water book of water book of water book of water book of' \
+                      'water book of water book of water book of water', comments_counter: 0, likes_counter: 0)
+
+    @cmt1 = Comment.create(user: @user2, post: @post2, text: 'Hey nice post!')
+    @cmt2 = Comment.create(user: @user2, post: @post2, text: 'Hey great post!')
```

Comment:
#### 🌟 Optional Suggestion 

- [ ] Consider organizing your feature tests within a feature folder. For each feature (e.g., Users), create a dedicated folder (e.g., users) and inside it, have `index`and `show` files to test corresponding views. This approach enhances code organization, separates concerns, and makes maintenance a breeze! 🚀

#### Example
```
📁 spec
  📁 features
    📁 users
      📄 index_spec.rb
      📄 show_spec.rb
    📁 posts
      📄 index_spec.rb
      📄 show_spec.rb


```

#### Keep up with the good work! 🌟✨